src/config/dynamic-theme-fixes.config

```diff
@@ -9866,6 +9866,7 @@ img[alt="Aksara Sunda.png"]
 img[alt="Aksara kawi name.png"]
 img[src*="/static/images/mobile/copyright/wikipedia-wordmark"]
 img[src*="/static/images/mobile/copyright/wikipedia-tagline"]
+[src="//upload.wikimedia.org/wikipedia/commons/thumb/e/ed/UML_diagrams_overview.svg/600px-UML_diagrams_overview.svg.png"]
```

Comment:
Shorting it to something like
```CSS
img[src*="UML_diagrams_overview.svg"]
```
Should be cleaner.

Reply:
Done 👍 c3954f3fa022d060ee8873b234c69b8ed2ec5de6